backend/translation/views.py

```diff
@@ -251,7 +298,7 @@ def generate_translation(request):
 
     # Get the query params
     transcript_id = request.query_params.get("transcript_id")
-    target_lang = request.query_params.get("target_lang")
+    target_language = request.query_params.get("target_language")
     batch_size = request.query_params.get("batch_size", 75)
 
     # Ensure that the UUID is valid
```

Comment:
Can we please remove here as well, as discussed earlier, to ensure consistency with `video` and `transcript` apps.